docs/content/technical-specifications.md

```diff
@@ -106,8 +111,4 @@ You can customize an existing Dockerfile or use it as a reference to create a ne
 
 ## Supported Helm Versions
 
-The Ingress Controller supports installation via Helm 3.0+.
-
-## Recommended Hardware
-
-See the [Sizing guide](https://www.nginx.com/resources/datasheets/nginx-ingress-controller-kubernetes-sizing-guide/) for recommendations.
+NGINX Ingress Controller supports [installation via Helm 3.0+]({{< relref "/installation/installing-nic/installation-with-helm.md" >}}).
```

Comment:
```suggestion
NGINX Ingress Controller supports [installation] via Helm 3.0+({{< relref "/installation/installing-nic/installation-with-helm.md" >}}).
```

Reply:
I think that change won't work.
![image](https://github.com/nginxinc/kubernetes-ingress/assets/78599298/b598798d-e4d0-4b24-945f-80fa940ab998)

I am opening a separate PR to change that link , as this is now merged to the revamp branch

Reply:
ah sorry I meant to move the entire link to just `installation`